lib/carrierwave/sequel.rb

```diff
@@ -11,6 +11,43 @@ def mount_uploader(column, uploader, options={}, &block)
       raise "You need to use Sequel 3.0 or higher. Please upgrade." unless ::Sequel::Model.respond_to?(:plugin)
       super
 
+      mod = Module.new
+      include mod
+      mod.class_eval <<-RUBY, __FILE__, __LINE__+1
+        def #{column}=(new_file)
+          if !(new_file.blank? && send(:#{column}).blank?)
+            modified!
+          end
+          super
+        end
+
+        def remove_#{column}!
```

Comment:
@mrbrdo but then a second question: shouldn't we provide `store!` method? Why doesn't ActiveRecord's version do it? What's the difference from `remove!`? It [should](https://github.com/carrierwaveuploader/carrierwave/blob/master/spec/uploader/store_spec.rb#L66)) modify column and calls `write_#{column}_identifier`. Shouldn't?